models/professor.py

```diff
@@ -266,6 +268,12 @@ def get_departments_taught(self):
                 group(1).upper() for uc in ucs)
         return sorted(departments_taught)
 
+    def transfer_reviews_from(self, prof):
+        mcs = user_course.MenloCourse.objects(professor_id=prof.id)
```

Comment:
Yeah, I like the default parameter idea. Originally, I was going to call this function, then delete the duplicate prof, but I agree the default arg will act as a reminder. Good call.